homeassistant/components/neato/const.py

```diff
@@ -119,6 +119,7 @@
     "nav_backdrop_frontbump": "Clear my path",
     "nav_backdrop_leftbump": "Clear my path",
     "nav_backdrop_wheelextended": "Clear my path",
+    "nav_floorplan_zone_path_blocked": "Clear my path",
```

Comment:
I do not see this in the official neato docs which is what we are replicating.  How did you come to find this error?

https://developers.neatorobotics.com/api/robot-remote-protocol/request-response-formats

Everything in this list needs to be documented on this page

Reply:
My D7 got stuck. I explored state with pybotvac and stubbled upon it.
```py
>>> pprint(robot.state)
{'action': 11,
 'alert': None,
 'availableCommands': {'goToBase': False,
                       'pause': False,
                       'resume': False,
                       'start': False,
                       'stop': False},
 'availableServices': {'IECTest': 'advanced-1',
                       'findMe': 'basic-1',
                       'generalInfo': 'basic-1',
                       'houseCleaning': 'basic-4',
                       'logCopy': 'basic-1',
                       'manualCleaning': 'basic-1',
                       'maps': 'basic-2',
                       'preferences': 'basic-2',
                       'schedule': 'basic-2',
                       'softwareUpdate': 'basic-1',
                       'spotCleaning': 'basic-3',
                       'wifi': 'basic-1'},
 'cleaning': {'boundary': {'id': 'e841e843-892d-4499-996b-77fc0c3bf5a9',
                           'name': 'Entrée'},
              'category': 4,
              'mapId': '2021-03-08T07:34:50Z',
              'mode': 2,
              'modifier': 1,
              'navigationMode': 3,
              'spotHeight': 0,
              'spotWidth': 0},
 'data': {},
 'details': {'charge': 98,
             'dockHasBeenSeen': True,
             'isCharging': False,
             'isDocked': False,
             'isScheduleEnabled': True},
 'error': 'nav_floorplan_zone_path_blocked',
 'meta': {'firmware': '4.6.0-72', 'modelName': 'BotVacD7Connected'},
 'reqId': '1',
 'result': 'ok',
 'state': 4,
 'version': 1}
```

Reply:
wouldnt be the first time the neato docs were out of date to the actual API :)

Reply:
Oh you know what it looks like you have a later firmware update than I do and there are no updates available yet for my D7's.  Maybe they are getting ready to introduce this change. My firmware is on 4.5.3-189. Hopefully that is the only change

Reply:
Yup you got some unreleased/undocumented firmware 😂 

https://neatorobotics.com/my-neato/software-update-d7/